articles/azure-functions/durable/durable-functions-entities.md

```diff
@@ -19,28 +19,52 @@ Entity functions define operations for reading and updating small pieces of stat
 > [!NOTE]
 > Entity functions and related functionality is only available in Durable Functions 2.0 and above. Entity functions are currently in public preview.
 
-## Entity identity
+## General concepts
 
-Entities (sometimes referred to as entity *instances*) are accessed via a unique identifier, the *entity ID*. An entity ID is simply a pair of strings that uniquely identifies an entity instance. It consists of:
+Entities behave like tiny services that have an identity and a state. Entities can perform operations, which may update the internal state, and perform I/O (e.g. call external services). To prevent conflicts, all operations performed by the same entity are guaranteed to be executed serially, i.e. one after another. Entities provide opportunity for scale-out, if the work is distributed across many entities and the state stored within each entity is of modest size.
 
-* An **entity name**: a name that identifies the type of the entity (for example, "Counter").
+**Entity Id.** Entities are accessed via a unique identifier, the *entity ID*. An entity ID is simply a pair of strings that uniquely identifies an entity instance. It consists of:
+
+* An **entity name**: a name that identifies the type of the entity (for example, "Counter"). This name must match the name of the entity function that implements the entity. It is case-insensitive.
```

Comment:
The prefixing for these bullets is inconsistent. I suggest starting each of these bullets the same way.

Reply:
I am using two different formattings here... one is for paragraphs that start with a bold work. The other one is for bulleted lists (that are nested inside the paragraphs). I guess it's hard to see that in the markdown without the preview.